App crash while enabling and disabling coins

Crash log
```
 java.lang.RuntimeException: Unable to start activity ComponentInfo{io.horizontalsystems.bankwallet.dev.testnet/io.horizontalsystems.bankwallet.modules.pin.PinActivity}: kotlin.UninitializedPropertyAccessException: lateinit property cryptoObject has not been initialized
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2946)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3046)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1688)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6798)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:767)
     Caused by: kotlin.UninitializedPropertyAccessException: lateinit property cryptoObject has not been initialized
        at io.horizontalsystems.bankwallet.core.security.FingerprintAuthenticationDialogFragment.onViewCreated(FingerprintAuthenticationDialogFragment.kt:88)
        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1286)
        at android.app.FragmentManagerImpl.moveFragmentToExpectedState(FragmentManager.java:1549)
        at android.app.FragmentManagerImpl.moveToState(FragmentManager.java:1611)
        at android.app.FragmentManagerImpl.dispatchMoveToState(FragmentManager.java:3039)
        at android.app.FragmentManagerImpl.dispatchActivityCreated(FragmentManager.java:2991)
        at android.app.FragmentController.dispatchActivityCreated(FragmentController.java:178)
        at android.app.Activity.performCreateCommon(Activity.java:6997)
        at android.app.Activity.performCreate(Activity.java:7005)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1230)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2899)
```

--- a/app/src/main/java/io/horizontalsystems/bankwallet/core/security/FingerprintAuthenticationDialogFragment.kt
+++ b/app/src/main/java/io/horizontalsystems/bankwallet/core/security/FingerprintAuthenticationDialogFragment.kt
@@ -49,8 +49,8 @@ class FingerprintAuthenticationDialogFragment : DialogFragment(), FingerprintCal
     private lateinit var iconBackgroundImg: ImageView
     private lateinit var errorTextView: TextView
 
-    private lateinit var callback: Callback
-    private lateinit var cryptoObject: FingerprintManagerCompat.CryptoObject
+    private var callback: Callback? = null
+    private var cryptoObject: FingerprintManagerCompat.CryptoObject? = null
     private var authCallbackHandler: FingerprintAuthenticationHandler? = null
 
      private val ERROR_TIMEOUT_MILLIS: Long = 1900
@@ -85,7 +85,7 @@ class FingerprintAuthenticationDialogFragment : DialogFragment(), FingerprintCal
 
         cancelButton.setOnClickListener { dismiss() }
 
-        authCallbackHandler = FingerprintAuthenticationHandler(this, cryptoObject)
+        authCallbackHandler = cryptoObject?.let { FingerprintAuthenticationHandler(this, it) }
     }
 
     override fun onPause() {
@@ -122,7 +122,7 @@ class FingerprintAuthenticationDialogFragment : DialogFragment(), FingerprintCal
         }
         iconBackgroundImg.run {
             setImageTintColor(iconBackgroundImg, R.color.green_crypto)
-            postDelayed({ callback.onFingerprintAuthSucceed() }, SUCCESS_DELAY_MILLIS)
+            postDelayed({ callback?.onFingerprintAuthSucceed() }, SUCCESS_DELAY_MILLIS)
         }
     }
 

